docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md

```diff
@@ -1,25 +1,30 @@
 ---
 layout: tutorials
-permalink: /tutorials/create-jekyll-theme/
-title: Create your first Jekyll theme
+permalink: /tutorials/convert-existing-html-sites-to-jekyll-templates/
+title: Convert existing HTML sites to Jekyll Templates
 ---
 
 If you're looking for themes for your Jekyll site, you don't have to restrict yourself to existing Jekyll themes. It's pretty easy to convert almost any static HTML site into a Jekyll theme.
 
-In many ways, any site that is currently a static site *already* is a Jekyll site. Jekyll just allows you to automate parts of the site (like inserting pages into templates, rendering lists for navigation, generating feeds and sitemaps, and more) as it processes the files.
+In many ways, any site that is currently a static site is *already* a Jekyll site. Jekyll just allows you to automate parts of the site (like inserting pages into templates, rendering lists for navigation, generating feeds and sitemaps, and more) as it processes the files.
 
-Understanding how to convert any HTML site into a Jekyll website will open your world to many more options for Jekyll themes. Instead of searching online for "Jekyll themes," you can choose from the large variety of HTML templates for your site, quickly Jekyll-ize the HTML template as you need to, and build the output with Jekyll.
+Understanding how to convert any HTML site into a Jekyll template will open your world to many more options for Jekyll themes. Instead of searching online for "Jekyll themes," you can choose from the large variety of HTML templates for your site, quickly Jekyll-ize the HTML template as you need to, and build the output with Jekyll.
```

Comment:
@ashmaroli  What do you think of `Understanding how to convert HTML pages into Jekyll templates` ?